### PR TITLE
Full code review

### PR DESCRIPTION
**Pull request by lukaselmer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+THUMBOR_SECURITY_KEY=UJwHAZLsRejTyLI88lAriHL7xAXa6q0umiwwpPcP \
+ BACKEND_ASSET_PATH=ao3ief5j.cloudfront.com \
+ THUMBOR_PATH=thumbor.example.com \
+ DEBUG=True \
+ python server.py
+```
+
+* Create a backend and upload images (e.g. S3)
+* Make the images accessible (e.g. Cloudfront)
+** From this step, you will get the BACKEND_ASSET_PATH, e.g. ao3ief5j.cloudfront.com
```
Comment (diegosteiner) [inline README.md]: On github nested lists need to be indented with two spaces: https://help.github.com/articles/markdown-basics/#nested-lists

File: README.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+* Generate a THUMBOR_SECURITY_KEY, configure thumbor
+* Transform images calling the right url (/thumb/<path:config>/u/<path:uri>)
+** Example: /thumb/200x300/smart/u/images/logo.png
```
Comment (diegosteiner) [inline README.md]: On github nested lists need to be indented with two spaces: https://help.github.com/articles/markdown-basics/#nested-lists

**Comment by diegosteiner** [comment]
Well documented, seems PEP8 compliant, overall straight forward. Looks good! 
Installation worked as declared with two minor flaws:  
1. `pyenv install` had to be changed to `pyenv install 3.4.3`
2. `pyenv rehash` had to be run after `pip install`

other than that: mergeable

**Comment by lukaselmer** [comment]
Nice, thx for your inputs, done!

**Pull request closed without merging**